frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -13,17 +12,15 @@ describe("scenarios > admin > settings", () => {
     cy.signInAsAdmin();
   });
 
-  itOpenSourceOnly(
-    "should prompt admin to migrate to the hosted instance",
-    () => {
-      cy.visit("/admin/settings/setup");
-      cy.findByText("Have your server maintained for you.");
-      cy.findByText("Migrate to Metabase Cloud.");
-      cy.findAllByRole("link", { name: "Learn more" })
-        .should("have.attr", "href")
-        .and("include", "/migrate/");
-    },
-  );
+  it("should prompt admin to migrate to the hosted instance", () => {
+    cy.skipOn(!!Cypress.env("HAS_ENTERPRISE_TOKEN"));
```

Comment:
I was thinking of extracting this in a variable, but it will have to be exported and imported in every single file that needs it. The benefit of that approach is readability.
```javascript
// frontend/test/__support__/cypress_data.js
export const ee = !!Cypress.env("HAS_ENTERPRISE_TOKEN");

// foo.cy.spec.js
cy.skipOn(ee);
cy.onlyOn(ee);
```